integration/kubernetes/ro-volume-exp.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+#
+# Copyright (c) 2021 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+expect -c "
```

Comment:
I know it's trivial, but please can you document the parameters. Also, maybe ensure they are specified and have a value?